dozer-sql/src/pipeline/expression/execution.rs

```diff
@@ -205,12 +204,23 @@ impl ExpressionExecutor for Expression {
                 right,
             } => operator.evaluate(schema, left, right, record),
             Expression::ScalarFunction { fun, args } => fun.evaluate(schema, args, record),
+
+            #[allow(unused_variables)]
             Expression::PythonUDF {
```

Comment:
Would it be more readable to feature-flag the `Expression::PythonUDF` variant instead? That way if a user tries to use a Python UDF when the feature is not enabled, then it would return a proper error instead of killing the thread (through the `panic!()` in line 222)

Reply:
Makes sense. Changed. Thanks @auterium